core/core_bind.h

```diff
@@ -124,9 +124,9 @@ class OS : public Object {
 	static OS *singleton;
 
 public:
-	enum VideoDriver {
-		VIDEO_DRIVER_VULKAN,
-		VIDEO_DRIVER_OPENGL_3,
+	enum RenderingDriver {
+		RENDERING_DRIVER_VULKAN,
+		RENDERING_DRIVER_OPENGL_3,
```

Comment:
Personally I think it would look better as `_OPENGL3` instead of `_OPENGL_3`, but I'm fine with either :)

Reply:
Me too, to be honest. I keep seeing a double underscore (`OPENGL__3`) all the time. Would `GLES3` be an option or have we departed from the `ES` naming?

Reply:
I would go with ``RENDERING_DRIVER_OPENGL3`` We are trying to consistently use ``OpenGL3`` for user-facing areas and GLES3 internally.